### DISCUSSION TITLE
Where to enter commands???

### DISCUSSION THREAD

**Discussion by sgordon777** [joedevivo/vscode-circuitpython#93] [General]
The documentation for this extension says:

command run:
circuitpython.openSerialMonitor
circuitpython.selectSerialPort
circuitpython.closeSerialMonitor

Sorry maybe it's just me, but I have no idea where to "run" these commands...  I've tried every conceivable place, including the "terminal"

-I have my Pi-Pico plugged in with code.py open in the vscode editor, libs, and others showing in the mapped drive
-Mu-editor works just fine.
-I have a "Circuit python serial monitor" in the right paign of the terminal, which does absolutely nothing.
-I can send commands to/from the Pico with a terminal emulator (Putty).

Another things. The documentation says something about the extension "activating" when I pull a file from the USB drive. WHen I loaded code.py, I saw no hint of this "activation"

I've tried several times to re-install the extension to no avail. An incredibly frustrating experience

**Comment by zwass** [comment]
If you open a directory in VS Code that contains one of those files, the extension will activate.

<img width="800" alt="Screen Shot 2022-07-15 at 6 06 44 PM" src="https://user-images.githubusercontent.com/575602/179328472-da010682-0fc3-446c-8143-2700e1344f23.png">

Opening the root directory of my circuitpython drive seems to do it for me.
